api/RTCDataChannel.json

```diff
@@ -0,0 +1,1024 @@
+{
+  "api": {
+    "RTCDataChannel": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCDataChannel",
+        "support": {
+          "webview_android": {
+            "version_added": "29"
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": "29"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
```

Comment:
Ok, this is a tricky one.

We have to set several entries here AND using alternate_name, instead of the none.
(identation not correct)
[ 
  { 
     "alternate_name": "DataChannel"
     "version_added"; "22"
  },
  {
      "version_added":"24"
}
]